docs/deployer-tool.md

```diff
@@ -0,0 +1,24 @@
+# Deploy/Upgrade RP
+
+`deploy` directory contains artifacts for manual environment deployment.
+Production deployment and upgrade can be orchestrated using deployment tooling:
+`go run ./cmd/aro deployer`
+
+Utility is decoupled from `env` package and requires these variables to run:
+```
+AZURE_TENANT_ID - Required
+AZURE_CLIENT_SECRET - Required
+AZURE_CLIENT_ID - Required
+AZURE_SUBSCRIPTION_ID - Required
+AZURE_RP_PARAMETERS_FILE - Required, location of environment parameters file
+AZURE_RP_RESOURCEGROUP_NAME - Required, RP resource group name
+AZURE_RP_VERSION - overrides gitCommit version for the RP VMSS
+AZURE_DEPLOY_NSG - re-deploys pre-deploy phase
+```
+
+* Utility will NOT create ResourceGroup. It expects ResourceGroup to be created before.
+* If ran on existing ResourceGroup it will update corresponding resources, as per generators output.
```

Comment:
why not just create it?

Reply:
If we create it we need to provide location too :) fine eitherway